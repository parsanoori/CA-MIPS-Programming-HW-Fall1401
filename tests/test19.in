2
766185144
377294497