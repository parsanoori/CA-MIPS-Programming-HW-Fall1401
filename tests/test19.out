377294497
766185144